errmod.c

```diff
@@ -201,7 +201,7 @@ int errmod_cal(const errmod_t *em, int n, int m, uint16_t *bases, float *q)
             } else q[j*m+k] = q[k*m+j] = -4.343 * em->lhet[cjk<<8|aux.c[k]]; // all the bases are either j or k
         }
         /* clamp to greater than 0 */
-        for (k = 0; k < m; ++k) if (q[j*m+k] < 0.0) q[j*m+k] = 0.0;
+        for (k = 0; k < m; ++k) if (q[j*m+k] < 0.0f) q[j*m+k] = 0.0f;
```

Comment:
Out of curiosity, why is it you need to fix this assignment, but not the `-4.343 * em->...` line above?

Reply:
@jkbonfield good question, I just went with the warnings GCC gave me. Anyhow, I've dropped the commit changing this, seeing it's not in the public API and therefore won't further the actual point of this PR.